src/Core/Datatypes/DenseColumnMatrix.h

```diff
@@ -48,6 +48,12 @@ namespace Datatypes {
 
     explicit DenseColumnMatrixGeneric(size_t nrows = 0) : EigenBase(nrows) {}
 
+    DenseColumnMatrixGeneric(std::vector<T> vals) : EigenBase(vals.size())
+    {
+      for (long unsigned int i = 0; i < vals.size(); ++i)
+        (*this)[i] = vals[i];
+    }
```

Comment:
Consider a move constructor for this case too; it would need a unit test, but would be super efficient.